reference/cd-github.md

```diff
@@ -0,0 +1,92 @@
+---
+title: GitHub Workflow (Preview)
+---
+
+<style>
+    .img-bordered {
+        border: 2px solid rgba(0,0,0,0.15);
+    }
+    .move-right {
+        margin-left: 32px;
+    }
+    .move-right-small {
+        margin-left: 8px;
+    }
+</style>
+
+Pulumi's GitHub workflow integration is a feature which shows the results of Pulumi stack
```

Comment:
Rather than say "Pulumi's GitHub workflow integration", it seems simpler to just say "Pulumi's GitHub App".

Reply:
Sure, done.